<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQcAppPage</class>
 <widget class="QWidget" name="QQcAppPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQcAppPage</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1911</width>
     <height>732</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>45</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>2</y>
     <width>100</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>质控品名称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>10</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="default_button">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>813</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>默认质控</string>
   </property>
   <property name="buttons" stdset="0">
    <string>flat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backup_button">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>813</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>备用质控</string>
   </property>
   <property name="buttons" stdset="0">
    <string>flat</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1912</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>791</y>
     <width>1912</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1</width>
     <height>792</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1911</x>
     <y>0</y>
     <width>1</width>
     <height>792</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>59</y>
     <width>1912</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>1910</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="bks" stdset="0">
    <string>gray</string>
   </property>
  </widget>
  <zorder>tableView</zorder>
  <zorder>default_button</zorder>
  <zorder>backup_button</zorder>
  <zorder>line_h</zorder>
  <zorder>line_h_2</zorder>
  <zorder>line_v</zorder>
  <zorder>line_v_2</zorder>
  <zorder>line_h_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>comboBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
